components/progress-indicator/__tests__/progress-indicator.browser-test.jsx

```diff
@@ -316,26 +306,6 @@ describe('SLDSProgressIndicator: ', () => {
 			expect(item.text()).to.include('Progress:');
 			expect(item.text()).to.include('%');
 		});
-
-		it('renders assistive text for steps', function () {
-			const firstItem = this.wrapper
-				.find('.slds-progress')
-				.find('li')
-				.find('.slds-tooltip-trigger')
-				.find('.slds-button')
-				.find('.slds-assistive-text')
-				.first();
-			expect(firstItem.text()).to.include('tooltip label');
-
-			const secondItem = this.wrapper
-				.find('.slds-progress')
-				.find('li')
-				.find('.slds-tooltip-trigger')
-				.find('.slds-button')
-				.find('.slds-assistive-text')
-				.at(1);
-			expect(secondItem.text()).to.include('Step');
-		});
```

Comment:
Is there a reason this unit test was removed?

Reply:
`.slds-assistive-text` was removed.